examples/error-handling/error_handling.description

```diff
@@ -1,2 +1,2 @@
-// In Ballerina, error handling can be performed either by returning a value of type `error`, or by throwing an exception.
+// In Ballerina, error handling can be performed either by returning a value of type `error`, or by panicking an error.
```

Comment:
I think we may have to rephrase and elaborate a bit more considering that errors have changed significantly